docker-compose-dev.yml

```diff
@@ -6,5 +6,6 @@ services:
       - S3_EXTENSIONS_BUCKET_HOST=brave-core-ext-dev.s3.bravesoftware.com
```

Comment:
docker compose files go out since they have environment-specific data. we can keep a basic one for local dev

Reply:
docker-compose files will not be needed anymore. Just keeping here for reference

Reply:
let's take them out then since they're publicly visible